.gitignore

```diff
@@ -90,6 +90,6 @@ typings/
 # DynamoDB Local files
 .dynamodb/
 
-lib/
+# lib/
```

Comment:
For using git for node modules, I'd recommend putting these changes on a branch not tied to the PR and rebase these changes on that branch when you need to re-build the lib directory